whelk-core/src/main/resources/ext/marcframe.json

```diff
@@ -22170,9 +22170,119 @@
         }
       ]
     },
-    "430": {
-      "ignored": true,
-      "TODO": "map, alternativeForm a Work (like a structured altLabel)"
+    "430" : {
+      "aboutEntity": "?thing",
+      "addLink": "hasVariant",
+      "include": ["i2-nonfiling"],
```

Comment:
For all other X30 we are using the pattern titledetails but obviously we can not use it for 430 as mapping of several properties differs. I think it would be a good idea to update titledetails to be able to be used for all X30 fields by grouping links/properties that still are the same, OR just add an TODO/NOTE that we are aware that 430 are different for $o, $s and $k and all X30 will be aligned.

Reply:
Absolutely, I will create new issue for refactoring as well.